bottlecap/src/telemetry/client.rs

```diff
@@ -19,7 +19,7 @@ impl TelemetryApiClient {
 
     pub async fn subscribe(&self) -> Result<Response, Box<dyn Error>> {
         let url = base_url(TELEMETRY_SUBSCRIPTION_ROUTE)?;
-        let resp = reqwest::Client::new()
+        let resp = reqwest::Client::builder().no_proxy().build().unwrap()
```

Comment:
is the client building a time consuming operation? If so, the client created in main could be reused

Reply:
it's pretty fast but yeah they do create a pool and it could be shared or reused